src/quest_manager/views.py

```diff
@@ -716,7 +724,10 @@ def complete(request, submission_id):
                         and submission.quest.specific_teacher_to_notify not in request.user.profile.current_teachers():
                     affected_users = [submission.quest.specific_teacher_to_notify, ]
                 else:
-                    affected_users = None
+                    # Send notification to current teacher when a quest is auto-approved
+                    if comment_text:
```

Comment:
Did you manually test this?  Because I think our summernote widget (the comment_text field widget) automatically adds `<p></br></p>` to every one of its fields (at least that's what you see when you are start editing text in one of the widget in html view.  Myabe summernote strips that if nothing is entered?  Just want to make sure before I accept the PR, sinse we don't have a unit test yet for this.

Reply:
You're right! `comment_text` always has a value of `(submitted without comment)`. I was wondering if it would be better to just check for the `form.cleaned_data['comment_text']` since it returns an empty string when there are no comments. Coz what I did what I just compared `comment_text != '(submitted without comment)'`